### PR TITLE
IDE-3704 Error creating fragment files

### PR DESCRIPTION
**Pull request by simonjhy**

### CODE REVIEW

File: tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/fragment/HostOSGiBundleDefaultValueService.java [hunk 1]
```diff
@@ -19,13 +19,13 @@
...
 import org.eclipse.core.resources.IProject;
-import org.eclipse.sapphire.DefaultValueService;
+import org.eclipse.sapphire.DerivedValueService;
 
 /**
  * @author Terry Jia
  * @author Andy Wu
  */
-public class HostOSGiBundleDefaultValueService extends DefaultValueService {
+public class HostOSGiBundleDefaultValueService extends DerivedValueService {
```
Comment (jtydhr88) [inline tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/fragment/HostOSGiBundleDefaultValueService.java]: @Camork what's different between **DerivedValueService** and **DefaultValueService**, why we need to switch to it??

Comment (Camork) [reply tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/fragment/HostOSGiBundleDefaultValueService.java]: This is a sapphire bug that the input field will be blank in linux when you use DefaultValueService ,and it's better to replace it in this wizard.

Comment (jtydhr88) [reply tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/fragment/HostOSGiBundleDefaultValueService.java]: what's action different between DerivedValueService and DefaultValueService

Comment (jtydhr88) [reply tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/fragment/HostOSGiBundleDefaultValueService.java]: if the DerivedValueService is better than DefaultValueService, we should open the ticket to replace all of them in our code or not just make change in only this ticket and only this file.

Comment (Camork) [reply tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/fragment/HostOSGiBundleDefaultValueService.java]: This situation is particular so i changed it by the way,but 'DerivedValueService' is not suitable for another wizard.

**Comment by jtydhr88** [comment]
sent to Greg, thanks.

**Pull request closed without merging**